jobs/release-monitoring/discovery/github/Jenkinsfile

```diff
@@ -114,31 +114,6 @@ def getLatestRelease(gitOrg, gitRepo, gitTokenId, releaseFetchMethod) {
     }
 }
 
-def getLabelScore(label) {
-    if (!label) {
-        return 0
-    }
-
-    def score = 0
-    def scores = [
-            'GA': 1,
-            'redhat': 1
-    ]
-    def parts = label.tokenize('-')
-
-    parts.each { part ->
-        if (scores.containsKey(part)) {
-            score += scores[part]
-        }
-
-        if (!scores.containsKey(part) && part.isInteger()) {
-            score += part.toInteger()
-        }
-    }
-
-    return score
-}
-
```

Comment:
Why is this not needed anymore? Not really sure what it was doing initially @odra added it. Think it was supposed to figure out if a tag indicated a new GA release or something.

Reply:
The way we're getting the [current/latest ](https://github.com/integr8ly/ci-cd/blob/master/jobs/release-monitoring/discovery/github/Jenkinsfile#L148-L152)version right now is by removing all text apart from the semver. This includes removing the label to indicate that it's a GA release. This function isn't doing anything for us at the moment because the [value](https://github.com/integr8ly/ci-cd/blob/master/jobs/release-monitoring/discovery/github/Jenkinsfile#L182) that's being passed to that function will always be null. 

We are [filtering](https://github.com/integr8ly/ci-cd/blob/master/jobs/release-monitoring/discovery/github/Jenkinsfile#L85) for the GA releases when we get releases by tags so I don't think we need that function at the moment.